magda-web-client/src/Components/Dataset/Add/Pages/DatasetAddEndPage.tsx

```diff
@@ -0,0 +1,67 @@
+import "./DatasetAddEndPage.scss";
+
+import giantTickIcon from "assets/giant-tick.svg";
+import draftIcon from "assets/format-active.svg";
+import printIcon from "assets/print.svg";
+import React from "react";
+
+type Props = {
+    datasetId: string;
+};
+
+// If you are not in preview mode
+export default function DatasetAddEndPage(props: Props) {
+    const draftPage = "/dataset/" + props.datasetId + "/details";
+    const viewDraft = () => {
+        window.location.href = draftPage;
+    };
+
+    const printDraft = () => {
+        return (window.location.href = draftPage + "?print=true");
+    };
+    return (
+        <div className="row">
+            <div className="col-sm-12 end-preview-page-1">
+                <div className="end-preview-container-1">
+                    <img src={giantTickIcon} className="giant-tick" />
+                    <h2>You're all done!</h2>
+                </div>
+                <br />
+                <div className="end-preview-container-2">
+                    <h3>
+                        Your dataset has been successfully sent off for
+                        approval.
+                    </h3>
+                    <br />
+                    <p className="dataset-status-txt">
+                        You can view the status of your dataset from{" "}
+                        <a href="/">your home page</a>.
+                    </p>
+                </div>
+                <br />
```

Comment:
Could you use a margin or padding css style for this? Doing layout like this was pretty common years ago, but it tends to get you in trouble if anyone changes the line-height.

Reply:
Styled with `margin-top`